config.py

```diff
@@ -23,6 +23,16 @@
             'unknown': settings.get('timeouts', {}).get('unknown', '6h'),
             'resolved': settings.get('timeouts', {}).get('resolved', '12h'),
         }
+        incident = {
+            'new_firing_alerts_notifications':
+                settings.get('incident', {}).get('new_firing_alerts_notifications', False),
+            'new_resolved_alerts_notifications':
+                settings.get('incident', {}).get('new_resolved_alerts_notifications', False),
+        }
+        experimental = {
+            'release_incident_and_recreate_chain_by_new_firing_alerts':
+                settings.get('experimental', {}).get('release_incident_and_recreate_chain_by_new_firing_alerts', False),
+        }
```

Comment:
Is there any sense to set experimental like this instead of:
```
experimental = settings.get('experimental', {})
```

Reply:
No, I think
```
experimental = settings.get('experimental', {})
```
is better :+1: